src/github_test.ts

```diff
@@ -140,3 +141,23 @@ Deno.test("fetchLastComment() returns the appropriate comment", async () => {
     ),
   );
 });
+
+Deno.test("backportPrExists() returns the appropriate result", async () => {
+  const prAndVersionToBackportExists = {
+    "30511_1.22": true,
+    "30511_1.21": true,
+    "30511_1.20": false,
+  };
+  await Promise.all(
+    Object.entries(prAndVersionToBackportExists).map(
+      async ([prAndVersion, backportExists]) => {
+        const [prNumber, version] = prAndVersion.split("_");
+        const result = await backportPrExists(
+          { number: Number(prNumber) },
+          version,
+        );
+        assertEquals(result, backportExists);
+      },
+    ),
```

Comment:
```suggestion
    Object.entries(prAndVersionToBackportExists).map(async ([prAndVersion, backportExists]) => {
      const [prNumber, version] = prAndVersion.split("_");
      const result = await backportPrExists(
        { number: Number(prNumber) },
        version,
      );
      assertEquals(result, backportExists);
    }),
```

prefer to put callback signature on same line, 97 chars line length is still ok with me.

Reply:
Why not commit? Do you have some code formatter ongoing?

Reply:
I committed it https://github.com/GiteaBot/gitea-backporter/pull/128/commits/e8187bdb9dff59dd822d78b724c00f36f57e55fd then ran deno fmt https://github.com/GiteaBot/gitea-backporter/pull/128/commits/ed9ee2ab7d62fff89c4d9326ecb97ec9f65cf3ea

Reply:
I see, so formatting nits are basically auto-rejected :D